### DISCUSSION TITLE
Bookmarks, observing Lists observations and comments

### DISCUSSION THREAD

**Discussion by Menno5** [Stellarium/stellarium#1939] [General]
Hello all

Original posting is now as Issue #1944 

But some observations and comments about the Observing Lists feature.
I do not know why the Bookmark feature is replaced by the Observing List. I'm sure it had good reasons.
But this is something that for a lot of users is totally unexpected behaviour.
This is a **major** change and from 1 to the other version, suddenly the Bookmarks are gone and replaced with Observing List.
Bookmarks were widely used, there are downloads on multiple sites where lists can be downloaded to import into the Bookmarks.
Other people (like me) have a big Bookmark list or in schools or public observatories, it's used by several users to add interesting objects.

I understand that it's in development but to implement it in it's current state and then bug-fix on the fly, is just not good at all. Actually it's a no-go for such a widely used feature and is a classic example where users can/will feel like they are Beta testers.
At least a temporary co-existence of Bookmarks and Observing List should have been implemented so people can get used.
Instead everybody who upgrades to 0.21.2 now is confronted with Bookmarks that are gone and bugs of a feature that still is in development.
And confronted with a new feature that lacks the functionality that made the Bookmarks so great: no adding of extra data as date and time to an object, no more names but only number designation of objects, use of abbreviation for the constellation (for a lot of Stellarium users this is very unclear since they only know about the full names of constellations).
Plus the Observing List lacks the intuitive use the Bookmarks had.
I say this because Stellarium is used by a very wide range of users: from the 10 year old boy/girl who discovers astronomy to the amateur users like me who do low key captures to (semi)professionals who use all the features of Stellarium. That last group maybe finds the Observing List logical but for the average user it's confusing to say the least in its current state.

I know this all maybe sounds harsh, I know a lot of time and effort is put into Stellarium by a small team. And I acknowledge that.
But this really breaks the use of Stellarium and should have been noted with Beta testing.
As I understand it, there are not enough Beta testers. So if needed I volunteer for that. Not as developer (zero skills in that field) but as normal, general user so things like this can be prevented or at least reduced in future releases.

Thanks in advance.

**Comment by axd1967** [comment]
Hello @Menno5 ... 

I've tried and tried to convince the team that the feature was not mature, to no avail. It's extremely frustrating to watch how this was totally predictable and went the wrong way, and we can only hope this never happens again. And this is all due to a single commit that was very simple to revert.

Mind you: the concept is good, i.e. we need to be able to manage **sets** of bookmarks. In fact that is the only change that was needed.

Now many people have been needlessly hurt. And this catastrophy could have been avoided.

You can find the entire discussion in #1816 .

What you can do is build a variant that continues to run the old bookmark lists (together with the new feature). Maybe my [patch](https://github.com/axd1967/stellarium/tree/alex/fork/bookmarks/old-bookmark-feature) can help so that anyone interested can continue to use the Bookmarks until the feature is ready. I intend to update this branch with the latest release, but it gives off a bitter taste. Normally the existing bookmarks are still usable with this patch, but to be on the safe side, make a copy of them before using this patch.

**Comment by Menno5** [comment]
Hi @axd1967

I can only look and comment on this from the user's side. Like I said, my developers skills are non-existent when it comes to coding and so on (your patch for example, is abracadabra for me :) ). As a "simple" user this change is just making me scratch my head and wondering why this change?
But I did read several discussions here and it was striking to me that there were indeed legitimate and very logical concerns about this all but still it was kinda forced to implement.
Also for "simple" users .... (no, lets call us Basic users :) ) ... also, for Basic users for example like I said: the abbreviation of the constellations is now weird/confusing/unknown (CMa is just a big ? for me) and also the absence of Localized Names of the object is not helping (NGC 1333 is just a number for me but the Embryo Nebula, that is saying something to me).

But yes, sets of bookmarks would have been a good solution. I my case as "simple" user, I would make the sets Spring, Summer, Autumn and Winter. I use that now (I reverted back to  0.21.1 for the time being) too but as exported Bookmarks.

But nuff said. It's done and I made my point. I just hope that the whole team keeps in mind that Stellarium is used in **all** layers of users, from the Basic's to the Pro's. That Stellarium is used for simply browsing the skies and getting to know the skies to complete platform to do almost everything with.
What seems logical from a purely astronomical view, is not logical for Basic users :)

**Comment by axd1967** [comment]
What seems acceptable to me as an emergency workaround is to re-activate and publish (eg 0.21.2.1)  the Bookmark feature pending the Observation lists.

I've pushed updates ([here](https://github.com/axd1967/stellarium/tree/alex/fork/bookmarks/old-bookmark-feature)) that now also contain this:
- prevent an accidental clearing of all bookmarks
- a more logical ordering of the buttons (see [discussion](https://github.com/Stellarium/stellarium/discussions/1812))

(Sorry for those that cannot compile and use that version)

**Comment by BlackApron** [comment]
First of all, I love Stellarium. From my perspective, is by far the best planetarium software available today.

But having said that, I totally agree... "Observing Lists" feels like an undercooked feature that has brought more problems than anything good.

Two things would make the "Observation List" feature acceptable:

1. To have an option to convert your bookmarks and import them into a list called "Bookmarks"
2. To have a description added to every object as the bookmarks option did... I just can't remember the catalog code for every object. Is a lot easier to look for "Cone Nebula" in my list rather than LDN1613.

Please either bring back the old Bookmarks or improve OL's... but quickly.

:)

**Comment by Menno5** [comment]
For those depending on their Bookmarks, this is not fixed in the new 21.3 version. As I understand it, this will be fixed with version 22.x, see #976 
For the time being just use version 21.1, this can be downloaded here: [https://github.com/Stellarium/stellarium/releases/tag/v0.21.1](https://github.com/Stellarium/stellarium/releases/tag/v0.21.1)
